src/components/NotificationsBanner.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+import { graphql, useStaticQuery } from 'gatsby'
+import get from 'lodash/get'
+import styled from '@emotion/styled'
```

Comment:
I see there are q couple of linting warnings, do you mind cleaning them up please?

Reply:
fixed. missed those.